### PR TITLE
update for chalk

### PR DESCRIPTION
**Pull request by ghost**
Updated example for chalk

### CODE REVIEW

File: src/app_mockup_helper.c [hunk 1]
```diff
@@ -15,7 +15,11 @@ static void main_window_load(Window *window) {
   Layer *window_layer = window_get_root_layer(s_main_window);
   GRect bounds = layer_get_frame(window_layer);
 
+#ifdef PBL_ROUND
```
Comment (C-D-Lewis) [inline src/app_mockup_helper.c]: Use `PBL_IF_ROUND_ELSE()` for single statements.

**Comment by C-D-Lewis** [comment]
Agree with @Neal re: more enticing example images per platform, but not a necessity IMO.

**Comment by ghost** [comment]
@Neal  @C-D-Lewis  New images for each platform have been added. Added platform specific resources so no need for checking if round or rect is defined.

**Comment by C-D-Lewis** [comment]
Aplite images should usually be pre-formatted for the black and white palette, but it appears to work. Up to you whether you want to pre-process the Aplite image (I would recommend it.) Aside from that, LGTM :+1:

**Comment by ghost** [comment]
@C-D-Lewis Image has been re-formatted to black and white for aplite. Merge to master?

**Comment by C-D-Lewis** [comment]
:ship:

**Pull request merged**